Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,14 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/archive/1.34.1.tar.gz"
+  sha256 "e0e97a94c80be8323772dc8953d463fe8eaf60c3a1c5f212d0b575a3b5c640ba"
+
+  depends_on :java
+  depends_on "maven" => :build
+
+  def install
+    system "mvn", "clean", "install"
+    libexec.install Dir["*"]
+    bin.write_jar_script libexec/"target/gcviewer-1.34.1.jar", "gcviewer"
```

Comment:
yeah but I figured that having a `version` variable without using it everywhere might just confuse things up so I dropped it, at least it's clear that it needs to be replaced manually when updating it...